src/partials/pages/_3d-modeling.html

```diff
@@ -182,7 +174,7 @@ <h3 class="models-image">
     </ul>
   </div>
 
-  <div class="modeling-btn">
+  <div class="modeling__btn">
```

Comment:
We don't need separate modeling__btn class at all - this property you give to it in styles is already within the default button component.

Liaise with Natasha to figure out, please.

Reply:
every button in pages (modeling, scanning and printing) have such style!!!

Reply:
see please

Reply:
I see, you gave the positioning for include through the div with .modeling__btn class.
It's half of a way, but leads to different solutions at all the pages.

The right way to resolve this bug is 
1) check the other pages button positioning wrappers, if they contain the same styles as yours (they do in our case)
2) create card in Trello to create the button wrapper in _base.scss styles and apply it to all the pages